base/commands/shell.go

```diff
@@ -80,6 +85,7 @@ func (cm *ShellCommand) ExecInteractive(ctx context.Context, ec plug.ExecContext
 			logLevel := strings.ToUpper(ec.Props().GetString(clc.PropertyLogLevel))
 			logText = fmt.Sprintf("Log %9s : %s", logLevel, logPath)
 		}
+		maybePrintNewerVersion(ec)
```

Comment:
Could you move this line after the following, so the update notification is displayed after the banner?